test/OfflineTest.php

```diff
@@ -80,6 +80,7 @@ public function testOptionsGetter()
                                  'useStartTls'            => false,
                                  'optReferrals'           => false,
                                  'tryUsernameSplit'       => true,
+                                 'reconnectAttempts'      => 0,
```

Comment:
Does that really need to be set? Because if there's no fallback and that needs to be set that is a Backwards incompatibility! And I'd like to avoid that!

Reply:
This shouldn't create a BC breakage. Ldap::getOptions() has always returned the current values of all supported options, but you don't have to pass anything new to the constructor or setOptions().